_drafts/circle-two-point-oh.md

```diff
@@ -0,0 +1,28 @@
+Notes from meeting in NYC:
+
+* We wanted to return early in the case of fast failure like rubocop and we
+  figured out how to hack it together in 1.0, but 2.0's workflows more closely
+  matched what we wanted to do.
+
+* start by describing the series of events.
+
+* do a gist of full config file, but also get the workflows part embedded.
+
+Random thoughts today:
+
+* don't feel like you have to explain everything about Circle CI 2.0 - it's ok
+  to link to their docs.
+
+* how do we talk about Volt externally? should i refer to it as our CMS for
+  galleries?
+
+* something like
+  [this](https://circleci.com/workflow-run/a2085d22-0737-4eaf-a363-666c483de278)
```

Comment:
I can't access this – probably a permissions thing. Want to grab this (and any other visuals) and throw them into the `_drafts` folder?

Reply:
Hmm for some reason, I cannot access the Workflows page on CircleCI, like https://circleci.com/gh/artsy/workflows/volt.

Reply:
Tracked here: https://discuss.circleci.com/t/cannot-load-workflows-request-uri-too-long/17889